frontend/src/pages/admin/users/interfaces/IUserRecord.tsx

```diff
@@ -0,0 +1,16 @@
+export enum AccountActive {
+  YES = 'Yes',
+  NO = 'No',
+}
+
+export interface IUserRecord {
```

Comment:
Should most of these properties be nullable?

Reply:
This interface is populated from the IUser interface and the values can be undefined/optional